src/proto/Binance.proto

```diff
@@ -220,4 +220,10 @@ message SigningInput {
 message SigningOutput {
     // Signed and encoded transaction bytes.
     bytes encoded = 1;
+
+    /// error code, 0 is ok, other codes will be treated as errors
+    int32 errorCode = 2;
```

Comment:
In proto files we use snake_case field names, please change for consistency.

Reply:
fixed